hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/WritableRatisContainerProvider.java

```diff
@@ -136,11 +137,16 @@ public ContainerInfo getContainer(final long size,
             } catch (IOException e) {
               LOG.warn("Waiting for one of pipelines {} to be OPEN failed. ",
                       allocatedPipelineIDs, e);
+              failureReason = "Waiting for one of pipelines to be OPEN failed. "
+                  + e.getMessage();
```

Comment:
Should we include the allocated pipeline IDs in this message too?

Reply:
Would it provide more clarity than the error message? It might not be efficient to send allocated pipeline list to the client is it is too large.

Reply:
Sure we can omit it.